addons/web/models/models.py

```diff
@@ -52,6 +52,18 @@ def web_search_read(self, domain=None, fields=None, offset=0, limit=None, order=
         }
         """
         records = self.search_read(domain, fields, offset=offset, limit=limit, order=order)
+        return self._format_web_search_read_results(domain, records, offset, limit, count_limit)
+
+    @api.model
+    def unity_web_search_read(self, domain=None, specification=None, offset=0, limit=None, order=None, count_limit=None):
```

Comment:
the default of limit shouldn't be '∞' instead ? `limit= '∞', order=None, count_limit= '∞'`






:trollface: